capstone/src/main/java/com/google/sps/data/dexparser/DexLoader.java

```diff
@@ -0,0 +1,112 @@
+package com.google.sps.data.dexparser;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.FileInputStream;
+
+import java.util.Scanner;
+import java.util.zip.ZipEntry;
+import java.util.zip.ZipInputStream;
+import java.util.logging.Level; 
+import java.util.logging.Logger; 
+import java.util.logging.*; 
+
+/*Initiates parsing of the DEX files
+ *
+ *This class can run on its own or its methods can be called 
+ */
+public class DexLoader {
+
+  private static final Logger LOGGER = Logger.getLogger(DexLoader.class.getName());
+
+  public static void main(String[] args) {
+
+    // To run this locally, input file path
+    System.out.print("Enter the file path of your APK: ");
+    Scanner input = new Scanner(System.in);
+
+    String filePath = input.next();
+
+    // Close scanner
+    input.close();
+
+    analyzeApkFeaturesLocally(filePath);
+  }
+
+
+  /*
+   * Wrapper used specifically for running function similar to unzipping locally for parsing
+   */
+  public static boolean analyzeApkFeaturesLocally(String filePath) {
+    long totalApkSize = 0;
+    long filesCount = 0;
+
+    try {
+
+      //Declare unzip elements
+      FileInputStream is = new FileInputStream(filePath);
+      ZipInputStream zis = new ZipInputStream(is);
+      ZipEntry ze = zis.getNextEntry();
+
+      while(ze != null) {
+        String fileName = ze.getName();
+        LOGGER.info("File Name: " + fileName);
```

Comment:
string formatting can be used

Reply:
Could you elaborate on it?

Reply:
LOGGER.log(Level.INFO,"Hello World {0}", "Param");
But that might be too much